Libraries/D3D12RaytracingFallback/src/TraverseFunction.hlsli

```diff
@@ -659,6 +659,8 @@ bool Traverse(
                                 instanceOffset);
```

Comment:
triMetadata should be unnecessary now, can you use primitiveMetadata in its place?